bin/abi_checker_module.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+
```

Comment:
This file should be added to `setup.py` under scripts. Perhaps `.py` would not be needed? Please see the example for `bin/ba_is_elf` 

Thanks

Reply:
Fixed.